README.md

```diff
@@ -3,7 +3,7 @@ Anserini
 ========
 [![build](https://github.com/castorini/anserini/actions/workflows/maven.yml/badge.svg)](https://github.com/castorini/anserini/actions)
 [![codecov](https://codecov.io/gh/castorini/anserini/branch/master/graph/badge.svg)](https://codecov.io/gh/castorini/anserini)
-[![Generic badge](https://img.shields.io/badge/Lucene-v9.3.0-brightgreen.svg)](https://archive.apache.org/dist/lucene/java/9.3.0/)
+[![Generic badge](https://img.shields.io/badge/Lucene-v9.3.0-brightgreen.svg)](https://archive.apache.org/dist/lucene/java/9.4.2/)
```

Comment:
shouldn't the badge link be https://img.shields.io/badge/Lucene-v9.4.2-brightgreen.svg ?

Reply:
Yea, good catch. thanks.